lib/build.gradle

```diff
@@ -4,25 +4,25 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.github.dcendents.android-maven'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 29
+        targetSdkVersion 30
         versionCode 24
         versionName "1.2.1"
     }
 }
 
 dependencies {
-    implementation "androidx.appcompat:appcompat:1.1.0"
-    implementation "com.google.android.material:material:1.1.0"
+    implementation "androidx.appcompat:appcompat:1.2.0"
+    implementation "com.google.android.material:material:1.2.1"
```

Comment:
Let's not bump this up, because dependent projects might not be able to use it. I think we should the minimum version that is API compatible with the latest

Reply:
- [ ] Rever `material` to 1.1.0

Reply:
Done

Reply:
Also reverted to appcompat 1.1.0